packages/components/Checkbox/src/Checkbox.types.ts

```diff
@@ -87,7 +88,8 @@ export interface ICheckboxTokens extends FontTokens, IForegroundColorTokens, IBa
 export interface ICheckboxSlotProps {
   root: React.PropsWithRef<IViewProps>;
   checkbox: IViewProps;
-  checkmark: ITextProps;
+  checkmark?: ITextProps;
+  checkmarkIcon?: IconProps;
```

Comment:
Could we still combine these slots into one? or is that not feasible?

Reply:
I think they can be merged, by introducing Platform.select() for the slot. But that would remove the strict check for the supplied props to the slot. Should I do that?

Reply:
I think we intentionally don't want the API to differ greatly between platforms. which is why I like the optional slots.  I was more thinking about how the image in the button could be a font, PNG, or SVG, yet there's only one slot it goes to and I was wondering if we could mimick that.

Reply:
The font icons in button are based on codepoint in a ttf file. Making that change here would unnecessarily complicate things. Also it would become tedious to keep track for different platforms. Keeping these separate with the hope that once SVG support comes to windows, we can have a single checkmark.

Reply:
OK, if it's much more complex to have them together, keeping separate makes sense! Thanks for the explanation =)